coalib/parsing/Globbing.py

```diff
@@ -29,6 +29,24 @@ def _end_of_set_index(string, start_index):
     return closing_index
 
 
+def glob_escape(string):
+    """
+    Escapes the given string with [c] pattern
```

Comment:
period at end missing.
Also I would not say "string" but "glob" (or "glob-string" if you want), since it operates on glob expressions, yeah "glob-expression" is even better :+1: